nix/nixos/cardano-exporter-options.nix

```diff
@@ -0,0 +1,115 @@
+{ config, lib, pkgs, ... }:
```

Comment:
the `mkMerge` and `environment` options are missing, and those helped simplify configuring things